src/pages/Homepage/components/Skeleton/SkeletonLineChart.tsx

```diff
@@ -0,0 +1,23 @@
+import ContentLoader from "react-content-loader";
+
+const SkeletonLineChart = () => {
+  return (
+    <ContentLoader
+      speed={4}
+      width="100%"
+      height={300}
+      backgroundColor="#f5f5f5"
+      foregroundColor="#dbdbdb"
+    >
+      <rect x="18" y="80" rx="20" ry="20" width="90%" height="180" />
+      <rect x="15" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="65" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="120" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="175" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="230" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="285" y="95%" rx="10" ry="10" width="35" height="15" />
+      <rect x="340" y="95%" rx="10" ry="10" width="35" height="15" />
+    </ContentLoader>
+  );
+};
```

Comment:
![image](https://user-images.githubusercontent.com/82322799/152011850-d062fe69-d520-4b6f-a5d3-c78d4192f242.png)


This is what I'm seeing. You may consider something consistent between all graph-cards , because it's looking a bit wired when you compare it to the Dounght beautiful skeleton.